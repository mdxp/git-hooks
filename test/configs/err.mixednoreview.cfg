[core]
	repositoryformatversion = 0
	filemode = true
	bare = false
	logallrefupdates = true
	ignorecase = true
[receive]
	denyCurrentBranch = ignore
[gate]
	name = gate
	shortname = app
	notify = invalidemailaddress@abadaddress
	user-check = skip
	bug-update = skip
	mail-debug = true
[gate "doc"]
	shortname = doc
	notify = differentinvalidemailaddress@abadaddress
	review-check = skip
[test]
	comment = 13408 remove mention of mirroring, raidz, and log devices
	files = doc/foo doc/bar appliance/server/foo
