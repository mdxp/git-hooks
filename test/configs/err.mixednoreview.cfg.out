remote: ERROR: bug 13408 is not covered by a pending review[K
